pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go

```diff
@@ -55,13 +55,15 @@ type singleRangeInfo struct {
 	token      rangecache.EvictionToken
 }
 
-var useDedicatedRangefeedConnectionClass = settings.RegisterBoolSetting(
```

Comment:
Is it safe to just remove a cluster setting, or there is some deprecation procedure?

Reply:
Done, thank you!